pom.xml

```diff
@@ -1508,18 +1439,37 @@
   </modules>
 
   <repositories>
-    <repository>
-      <id>kuali.nexus</id>
-      <name>Nexus Repository Manager</name>
-      <url>http://nexus.kuali.org/content/groups/public</url>
+      <repository>
+          <id>kuali.nexus</id>
+          <name>Nexus Repository Manager</name>
+          <url>http://nexus.kuali.org/content/groups/public</url>
+          <releases>
+              <enabled>true</enabled>
+          </releases>
+          <snapshots>
+              <enabled>false</enabled>
+          </snapshots>
+      </repository>
+      <repository>
+          <id>oss.sonatype.org</id>
```

Comment:
This additional repository was required to bring in the pre-release version of Eclipselink